recipe/meta.yaml

```diff
@@ -18,40 +18,53 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.7 #,<3.8
     - pip
   run:
     - python >=3.7
-    - click >=7.0
-    - cloudpickle >=1.3.0
-    - croniter >=0.3.24,<1.0
-    - dask-core >=2.17.0
-    - distributed >=2.17.0
-    - docker-py >=3.4.1
-    - importlib_resources >=3.0
-    - marshmallow >=3.0.0b19
-    - marshmallow-oneofschema >=2.0.0b2
-    - mypy_extensions >=0.4.0
-    - packaging >=20.0
-    - pendulum >=2.0.4
-    - python-dateutil >2.7.3
-    - pyyaml >=3.13
-    - python-box >=5.1.0
-    - python-slugify >=1.2.6
-    - pytz >=2018.7
-    - requests >=2.20
-    - tabulate >=0.8.0
-    - toml >=0.9.4
-    - urllib3 >=1.24.3
+    - aiofiles >=0.7.0
+    - aiohttp >=3.8.1
+    - aiosqlite >=0.17.0 
+    - alembic >=1.7.5
+    - anyio >=3.4.0
+    - asgi-lifespan >=1.0
+    - asyncpg >=0.23
+    - click >=8.0, < 8.2
+    - cloudpickle >=2.0
+    - coolname >=1.0.4
+    - croniter >=1.0.12
+    - cryptography >=36.0.1
+    - docker-py >=4.0
+    - fastapi >=0.70
+    - fsspec >=2022.5.0
+    - griffe >=0.20.0, <0.22.0 
+    - httpx >=0.22
+    - importlib_metadata >=4.4 #,python_version < 3.10
```

Comment:
Was not able to find a way to encode a specific python version for a package, however, we can encode a specific package version found on conda

Reply:
No luck with `- importlib_metadata >=4.4 # [py<=39]` ? That what the docs say and others have used these specifiers on dependencies (such as the matplotlib feedstock [here](https://github.com/conda-forge/matplotlib-feedstock/blob/da0e5171ab15212e095f60615d6e0cb13bdc2b66/recipe/meta.yaml#L52))

note, I use <=39 instead of <310 in order to avoid a bug in conda that struggle to read 310 as 3.10